openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -153,10 +153,18 @@
         also the responsibility of the centralized authentication
         service.
 
-##
-## Ref: https://access.redhat.com/solutions/2191161
-## "How to modify console session timeouts in Openshift 4"
-##
+        OpenShift automatically audits creation of the Identity
```

Comment:
@JAORMX Do you remember if we support custom audit profiles for the apiserver? I think the answer is 'no', but I wanted to doublecheck.

Reply:
We don't support custom profiles, no.

Reply:
great, then we don't need to create a rule as this would be inherently met (I checked that all three audit profiles would cover this control)

Reply:
We already have a rule for this anyway: https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/logging/audit_profile_set/rule.yml